README.md

```diff
@@ -100,7 +100,47 @@ Compile Sass as changes are made:
 
 ### Deployment
 
-See directions in the 18F/openFEC repo.
+##### Likely only useful for 18F team members
+
+Before deploying, install the [Cloud Foundry CLI](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) and the [autopilot plugin](https://github.com/concourse/autopilot):
+
+```
+cf install-plugin autopilot -r CF-Community
+```
+
+To deploy to Cloud Foundry, run `invoke deploy`. The `deploy` task will attempt to detect the appropriate
+Cloud Foundry space based the current branch; to override, pass the optional `--space` flag:
+
+```
+invoke deploy --space dev
+```
+
+The `deploy` task will use the `FEC_CF_USERNAME` and `FEC_CF_PASSWORD` environment variables to log in.
+If these variables are not provided, you will be prompted for your Cloud Foundry credentials.
+
+Credentials for Cloud Foundry applications are managed using user-provided services labeled as
+"fec-creds-prod", "fec-creds-stage", and "fec-creds-dev". Services are used to share credentials between the API and the webapp. To set up a service:
+
+```
+cf target -s dev
+cf cups fec-creds-dev -p '{"SQLA_CONN": "..."}'
+```
+
+To stand up a user-provided credential service that supports both the API and the webapp, ensure that
+the following keys are set:
+
+* SQLA_CONN
+* FEC_WEB_USERNAME
```

Comment:
Ah good to know, I wasn't aware of that.  I had taken what's currently in the API README for now just to get something in place given that the instructions had changed.  @emileighoutlaw are in the midst of updating that documentation but I'll be sure that we incorporate this information in those updates as well as reflecting them here.  There's no sense in perpetuating incorrect info, so thanks for flagging this!